### ISSUE TITLE
Specs: `attributes_for(...).except("created_at", "updated_at")` not necessary?

### ISSUE THREAD

**Issue by vfonic** [graphiti-api/graphiti#129]
Hey guys! Great gem! Thank you so much for building this!

I'm just diving into the gem, following the Quickstart guide.

I've been playing with the generated specs and found out that it's unnecessary to have the `.except("created_at", "updated_at")` part.

More specifically, it led me to write this:
`.except("created_at", "updated_at", "title")` which failed, because the only available keys in the post factory (from Quickstart) are `%i[title upvotes active]`:

```rb
FactoryBot.define do
  factory :post do
    title { "MyString" }
    upvotes { 1 }
    active { false }
  end
end
```

```rb
pry> attributes_for(:post)
=> {:title=>"MyString", :upvotes=>1, :active=>false}
pry> attributes_for(:post).except("title")
=> {:title=>"MyString", :upvotes=>1, :active=>false}
pry> attributes_for(:post).except(:title)
=> {:upvotes=>1, :active=>false}
```

**Comment by richmolj** [comment]
Hey @vfonic - thanks for the kind words ❤️ !

This is a good point; I think you're right they can be removed. @sideshowbandana created the original PR for this - Kyle, [I see you had a comment that attributes_for contained created/updated_at](https://github.com/graphiti-api/graphiti/pull/52#issuecomment-456121864) but it seems like that's not the case. Are you good with removing these?

**Comment by sideshowbandana** [comment]
> Are you good with removing these?

Sure. Here you go: https://github.com/graphiti-api/graphiti/pull/131

 I think it's a relic from switch back and forth from `build(..).attributes` to `attributes_for`. 

The former returns a hash with keys as strings along with "created_at", "updated_at" and any dependent foreign keys. The latter returns just the attributes with keys as symbols.

```
2.6.1 :001 > FactoryBot.build(:post).attributes.except("created_at", "title")
   (0.1ms)  BEGIN
  User Create (1.2ms)  INSERT INTO "users" ("created_at", "updated_at") VALUES ($1, $2) RETURNING "id"  [["created_at", "2019-04-26 05:37:54.694869"], ["updated_at", "2019-04-26 05:37:54.694869"]]
   (0.9ms)  COMMIT
 => {"id"=>nil, "upvotes"=>1, "active"=>false, "updated_at"=>nil, "user_id"=>"b1a94aec-a29d-49d1-b431-b754a6935ab9"} 
2.6.1 :002 > FactoryBot.attributes_for(:post).except(:title)
 => {:upvotes=>1, :active=>false} 
```

**Comment by vfonic** [comment]
Thank you @sideshowbandana for addressing this so quickly!

**Comment by sideshowbandana** [comment]
No problem. Happy to help

**Comment by richmolj** [comment]
Merged #131 to master, likely a new release this weekend. Thank you @vfonic for raising the issue and @sideshowbandana for the quick fix! 👏 🎉
